src/components/OSCALSystemImplementation.js

```diff
@@ -112,6 +113,11 @@ export default function OSCALSystemImplementation(props) {
               </TableContainer>
             </Grid>
           </Grid>
+          <OSCALInventoryItems
+            inventoryItems={props.systemImplementation["inventory-items"]}
+            parties={props.parties}
+            components={props.components}
+          />
```

Comment:
Is there a reason `InventoryItems` are both in `SSP` and `SystemImplementation`? I feel it makes more sense to just have it be part of `SystemImplementation` as that is also what the spec has.

Reply:
Where does this appear in `SSP`? It should only be in `SystemImplementation`.

Reply:
I think it was a weird viewing thing on my end. You are right.